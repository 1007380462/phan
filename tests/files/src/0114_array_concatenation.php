<?php
function f(array $v) {}
f([1, 2] + [2, 3]);

function g(int $p) {}
g([1, 2] + ['foo', 'bar']);
g([1, 2] + [3]);

