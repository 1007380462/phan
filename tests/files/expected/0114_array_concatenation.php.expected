%s:6 PhanTypeMismatchArgument Argument 1 (p) is array but \g() takes int defined at %s:5
%s:7 PhanTypeMismatchArgument Argument 1 (p) is int[] but \g() takes int defined at %s:5

